web/menu.jspf

```diff
@@ -92,7 +92,9 @@ org.opensolaris.opengrok.web.Util"
     <select tabindex="8" class="q" id="project"
         name="project" multiple="multiple" size="<%=
         Math.min(15, projectsSize) %>"><%
-        SortedSet<String> pRequested = cfg.getRequestedProjects();
+		SortedSet<String> pRequested = cfg.getRequestedProjects();
+		if (request.getParameter("hdnSearchAll") != null && request.getParameter("hdnSearchAll").equals("true"))
+			pRequested.clear(); 
```

Comment:
explain in a code comment why this is necessary so that someone in the future does not remove it impetuously